core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -174,7 +174,8 @@ public long getEstimatedSize()
                 currentPageBuilder.getRetainedSizeInBytes() +
                 sizeOf(groupIdsByHash) +
                 sizeOf(rawHashByHashPosition) +
-                preallocatedMemoryInBytes;
+                preallocatedMemoryInBytes +
+                ((dictionaryLookBack != null) ? dictionaryLookBack.getRetainedSizeInBytes() : 0);
```

Comment:
nit: Are all parentheses necessary?

Reply:
No, I've added clarifying parentheses for a tested expression.